sdk/cmake/gcc.cmake

```diff
@@ -280,13 +280,18 @@ function(set_image_base MODULE IMAGE_BASE)
 endfunction()
 
 function(set_module_type_toolchain MODULE TYPE)
-    if((${TYPE} STREQUAL "kernelmodedriver") OR (${TYPE} STREQUAL "wdmdriver"))
-        add_target_link_flags(${MODULE} "-Wl,--exclude-all-symbols,-file-alignment=0x1000,-section-alignment=0x1000")
+    if (TYPE IN_LIST KERNEL_MODULE_TYPES)
+        target_link_options(${MODULE} PRIVATE -Wl,--exclude-all-symbols,-file-alignment=0x1000,-section-alignment=0x1000)
         if(${TYPE} STREQUAL "wdmdriver")
-            add_target_link_flags(${MODULE} "-Wl,--wdmdriver")
+            target_link_options(${MODULE} PRIVATE "-Wl,--wdmdriver")
         endif()
-        #Disabled due to LD bug: ROSBE-154
-        #add_linker_script(${MODULE} ${REACTOS_SOURCE_DIR}/sdk/cmake/init-section.lds)
+        # Place INIT section at the tail of the module
+        add_linker_script(${MODULE} ${REACTOS_SOURCE_DIR}/sdk/cmake/init-section.lds)
+        # Fixup section characteristiscs
+        add_custom_command(TARGET ${MODULE} POST_BUILD
```

Comment:
I think it makes sense to leave a comment here about which characteristics are being added, so people don't need to look inside pefixup

Reply:
Sure